### DISCUSSION TITLE
Add support for marking chat unread

### DISCUSSION THREAD

**Discussion by abrac** [d99kris/nchat#178] [Ideas]
Is it possible to mark a chat as unread? I sometimes open a chat by mistake, and realise that I don't have time to respond to it immediately. So I want to mark it as unread, so that I can respond to it later.

**Comment by d99kris** [comment]
Hi @abrac - sorry this is not supported currently and it's not in the short-term plans. But if you like we can change this discussion thread to a feature request (changing Category to `Ideas`) and perhaps we'll come to it eventually.

**Reply by d99kris** [reply]
Just to add on - until such feature gets implemented - one can configure nchat to not automatically mark chats as read by setting `mark_read_on_view=0` in `~/.nchat/ui.conf`. It then requires you to press `page down` or `end` or send a message in that chat, for it to be marked read.

**Reply by abrac** [reply]
Thank you so much! That setup works perfectly for me!
